src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.test.ts

```diff
@@ -49,82 +37,83 @@ describe('addAggregatorAndDestinationToMemo', () => {
       finalAssetAddress: FINAL_ASSET_ADDRESS,
       minAmountOut,
       slippageBps,
-      finalAssetPrecision: 18,
-      chainNamespace: CHAIN_NAMESPACE.Evm,
+      longtailTokens: DEFAULT_THORCHAIN_ASSETS_LIST,
     })
 
     expect(modifiedMemo).toBe(
-      `=${MEMO_PART_DELIMITER}e${MEMO_PART_DELIMITER}${RECEIVE_ADDRESS}${MEMO_PART_DELIMITER}${expectedL1AmountOut}${MEMO_PART_DELIMITER}ss${MEMO_PART_DELIMITER}${affiliateBps}${MEMO_PART_DELIMITER}${AGGREGATOR_TWO_LAST_CHARS}${MEMO_PART_DELIMITER}${FINAL_ASSET_ADDRESS}${MEMO_PART_DELIMITER}206301621786412`,
+      `=${MEMO_PART_DELIMITER}e${MEMO_PART_DELIMITER}${RECEIVE_ADDRESS}${MEMO_PART_DELIMITER}${expectedL1AmountOut}${MEMO_PART_DELIMITER}ss${MEMO_PART_DELIMITER}${affiliateBps}${MEMO_PART_DELIMITER}${AGGREGATOR_TWO_LAST_CHARS}${MEMO_PART_DELIMITER}${SHORTENED_FINAL_ASSET_ADDRESS}${MEMO_PART_DELIMITER}941367142801`,
     )
   })
 
-  it('should throw if chainNamespace is UTXO and chainId is not BCH', () => {
+  it('should throw if chainId is BCH and bytes length is > 220', () => {
     const minAmountOut = '2083854765519275828179229'
-    const affiliateBps = '100'
-    const expectedL1AmountOut = '42' // we don't care about this for the purpose of tests
-    const quotedMemo = `=:ETH.ETH:${RECEIVE_ADDRESS}:${expectedL1AmountOut}:ss:${affiliateBps}:${AGGREGATOR_TWO_LAST_CHARS}:${FINAL_ASSET_ADDRESS}:10`
-
-    const slippageBps = 100 // 1%
+    const memoOver220Bytes = `=:ETH.ETH:${REALLY_BIG_ADDRESS}:${expectedL1AmountOut}:ss:${affiliateBps}`
 
     expect(() =>
       addAggregatorAndDestinationToMemo({
         sellChainId: dogeChainId,
-        quotedMemo,
+        quotedMemo: memoOver220Bytes,
         aggregator: AGGREGATOR_ADDRESS,
         finalAssetAddress: FINAL_ASSET_ADDRESS,
         minAmountOut,
         slippageBps,
-        finalAssetPrecision: 18,
-        chainNamespace: CHAIN_NAMESPACE.Utxo,
+        longtailTokens: DEFAULT_THORCHAIN_ASSETS_LIST,
       }),
     ).toThrow()
   })
 
-  it('should throw if chainId is BCH and bytes length is > 220', () => {
+  it('should throw if chainId is BTC and bytes length is > 80', () => {
     const minAmountOut = '2083854765519275828179229'
-    const affiliateBps = '100'
-    const expectedL1AmountOut = '42' // we don't care about this for the purpose of tests
-    const memoOver220Bytes = `=:ETH.ETH:${RECEIVE_ADDRESS}:${expectedL1AmountOut}:ss:${affiliateBps}:${AGGREGATOR_TWO_LAST_CHARS}:${FINAL_ASSET_ADDRESS}:${[
-      ...Array(220),
-    ].map((_, counter) => counter)}`
+    const memo = `=:ETH.ETH:${REALLY_BIG_ADDRESS}:${expectedL1AmountOut}:ss:${affiliateBps}`
```

Comment:
Same comment on this guy re: assert length or add a comment so that someone reading this knows this is actually the length it states